<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="33" total="34" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="72828" started-at="2020-08-17T18:00:26Z" finished-at="2020-08-17T18:01:39Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="72828" started-at="2020-08-17T18:00:26Z" finished-at="2020-08-17T18:01:39Z">
      <class name="Test.AudienceTest">
        <test-method status="PASS" signature="setExtent()[pri:0, instance:Test.AudienceTest@1b083826]" name="setExtent" is-config="true" duration-ms="187" started-at="2020-08-17T18:00:26Z" finished-at="2020-08-17T18:00:26Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method status="PASS" signature="setUp()[pri:0, instance:Test.AudienceTest@1b083826]" name="setUp" is-config="true" duration-ms="20682" started-at="2020-08-17T18:00:26Z" finished-at="2020-08-17T18:00:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setUp -->
        <test-method status="FAIL" signature="validateTC_AI_074()[pri:15, instance:Test.AudienceTest@1b083826]" name="validateTC_AI_074" duration-ms="49354" started-at="2020-08-17T18:00:47Z" finished-at="2020-08-17T18:01:37Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Alumni template is downloaded expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Alumni template is downloaded expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at Test.AudienceTest.validateTC_AI_074(AudienceTest.java:885)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTC_AI_074 -->
        <test-method status="PASS" signature="tearDown(org.testng.ITestResult)[pri:0, instance:Test.AudienceTest@1b083826]" name="tearDown" is-config="true" duration-ms="1944" started-at="2020-08-17T18:01:37Z" finished-at="2020-08-17T18:01:38Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validateTC_AI_074 status=FAILURE method=AudienceTest.validateTC_AI_074()[pri:15, instance:Test.AudienceTest@1b083826] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:Test.AudienceTest@1b083826]" name="endReport" is-config="true" duration-ms="626" started-at="2020-08-17T18:01:38Z" finished-at="2020-08-17T18:01:39Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- Test.AudienceTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
